src/leed_nsym/cleed_mode.c

```
@@ -0,0 +1,173 @@
+  if (*p == '\0' || cleed_is_comment_start(*p))
+  {
+    return 0;
+  }
+
+  return cleed_is_symmetry_token(p);
+}
+
+int cleed_detect_symmetry_file(const char *path, int *has_symmetry)
+{
+  FILE *stream = NULL;
+  char line[512];
```

Clang (14.0.6)

[warning] 512 is a magic number; consider replacing it with a named constant

(cppcoreguidelines-avoid-magic-numbers,readability-magic-numbers)